documents/operating.md

```diff
@@ -41,6 +43,7 @@ BIFFUD shall be dissolved as a Limited Liability Company if any of the following
 2. The Hive Mind fails to hold a single meeting with quorum for a period of one consecutive year.
 3. Everyone dies/there are no more Corporate Overlords.
 4. If all Corporate Overlords identify as cis-hetero white men.
+5. There are now COWS left in the ranks of the Corporate Overlords.
```

Comment:
How "now," (brown) COW?

Reply:
This seems like it should replace rule 3, right? This is just a more specific version of "no more Corporate Overlords" since now it has to be "no more Corporate Overlords That Are Legally Owners of the Company."